tests/acceptance/live-test.js

```diff
@@ -1,12 +1,12 @@
 import { module, test } from 'qunit';
-import { visit, currentURL, waitFor, typeIn, click } from '@ember/test-helpers';
+import { visit, currentURL, typeIn, click } from '@ember/test-helpers';
 import { setupApplicationTest } from 'website-www/tests/helpers';
 
 module('Acceptance | live', function (hooks) {
   setupApplicationTest(hooks);
   // TODO - remove test for dev part when it goes to production
   test('skeleton and video should exists after join modal', async function (assert) {
-    assert.expect(11);
+    assert.expect(6);
```

Comment:
Could you explain more about change in this test case

Reply:
https://github.com/Real-Dev-Squad/website-www/pull/498#discussion_r1240623617